.meta/sinks/papertrail.toml

```diff
@@ -0,0 +1,34 @@
+[sinks.papertrail]
+title = "Papertrail"
+beta = true
+common = false
+delivery_guarantee = "best_effort"
+egress_method = "streaming"
+function_category = "transmit"
+healthcheck = true
+input_types = ["log"]
+write_to_description = "[Papertrail](https://www.papertrail.com/)"
+
+<%= render("_partials/_component_options.toml", type: "sink", name: "papertrail") %>
+
+<%= render("_partials/_buffer_options.toml", namespace: "sinks.papertrail.options") %>
+
+[sinks.papertrail.options.endpoint]
+type = "string"
+common = true
+required = true
+examples = ["logs.papertrailapp.com:12345"]
+description = "The endpoint to stream logs to."
+
+[sinks.papertrail.options.encoding]
+type = "string"
+category = "requests"
+common = true
+required = true
```

Comment:
I tested with `encoding = "text"` and this sink doesn't work. Testing with `encoding = "json"` works fine.

Consider removing Text support from this sink.

Reply:
I just tested this and it works as expected? I don't see why we shouldn't support text since we are using the syslog protocol, its "message" field is designed to be not encoded.

Can you explain what issue you ran into?

Reply:
You're right... wtf? Why did my first 4 messages get lost then?